script/Ciel/Day 7/07_01_CIEL07_1B.txt

```diff
@@ -470,36 +487,38 @@ Whereas Ciel-senpai starts heading toward the third-year classroom on the second
 [sha:2cb4ec0b13948c511e4256187cf780372b98f5aa]{
 -- Page 69, Offset 14970.
 -- 詳しい話は放課後、茶道室で！」
-We can have a proper chat in the tea ceremony room once classes are over!"
+Once classes are over, we can pick up again in the tea ceremony room!"
```

Comment:
Please keep this and Offset 14965 the same: they get informally referenced later, and they're intentionally super similar.

Reply:
I think doing the thing where you just like list facts about an appointment would work. I cannot even begin to describe what this pattern is called in english but it's like:
"After school! Tea ceremony room! Let's have a proper chat" or something